api/V1/recordingUtil.js

```diff
@@ -187,12 +188,13 @@ async function report(request) {
       r.Group.groupname,
       r.Device.devicename,
       moment(r.recordingDateTime)
-        .tz("Pacific/Auckland")
+        .tz(config.timeZone)
         .format("YYYY-MM-DD"),
       moment(r.recordingDateTime)
-        .tz("Pacific/Auckland")
+        .tz(config.timeZone)
         .format("h:mm a"),
```

Comment:
What about the seconds? 

You could just go with 24hr time here, e.g. 14:06:11